### ISSUE TITLE
plot,ctd-method has incomplete documentation for which= possibilities

### ISSUE THREAD

**Issue by richardsc** [dankelley/oce#1292]
I noticed looking at the help page for `plot,ctd-method` that it omits documentation for quite a few of the possible `which=` values. In particular, I noticed the deficiency because I created a `ctd` object with `deploymentType='moored'`, for which the default plot includes T and S time series and a TS plot.

Looking at the documentation I see:
```
          If 'which' is not
          supplied, a default will be used. This default will be
          'c(1,2,3,5)' if the CTD is in profiling mode (i.e. if
          'deploymentType' in the 'metadata' slot equals '"profile"',
          or is missing) or '"thermosalinograph"', the default will be
          'c(30, 3, 31, 5)'.  If it is '"towyo"', 'c(30, 31, 32, 3)'
          will be used. Details are as follows.
```

1. Note that a description of what to use for `deploymentType='moored'` is not in there.

2. What do `which=30, 31, 32,` etc correspond to? The documentation of `which` types only goes to `which=21` (also, what about which=22..29?)

**Comment by richardsc** [comment]
Fixed in commit 10964bcddb. I meant to make a PR, but ended up making the changes on `develop` so just pushed. Sorry.

**Comment by dankelley** [comment]
Thanks -- I never would have noticed this, unless I was doing one of the graphs. I had a quick look at the diff, and it seems great. No worries on pushing to `develop`.  I always do a `git pull` when I start working on any branch of any project, and so the only time this sort of thing might be a problem would be if we were both working on a given complex part of code, which is unlikely to happen if we use issues to indicate that some code is going "hot" for a given author.
